index.html

```diff
@@ -87,7 +87,11 @@ <h1 class="logo-brand" itemprop="brand">MACILE</h1>
       </div>
     </div>
   </nav>
-
+  <div class="donate">
+      <h2>Support our growth!</h2>
```

Comment:
Let's use the custom font for the `h2`. The rest of the text will inherit `Roboto`.